common/Session.cpp

```diff
@@ -254,13 +254,15 @@ void Session::handleMessage(const std::vector<char> &data)
 {
     try
     {
+#if !MOBILEAPP
```

Comment:
Soo ... I suggest that we guard these UnitBase.get() calls with:

UnitBase::isUnitTesting() && ...

And then I think we don't get any crashes; and we don't have any extra compile conditionals =)

Reply:
OK. That founds like a reasonable approach. I'll see if I can implement that over the next day or two.

Is it safe to assume that any UnitWSD calls can also be guarded with UnitBase::isUnitTesting()?

Reply:
Done